libparsec/crates/client/src/client/mod.rs

```diff
@@ -44,7 +45,11 @@ use crate::{
 };
 use libparsec_client_connection::AuthenticatedCmds;
 use libparsec_platform_async::lock::Mutex as AsyncMutex;
+
 use libparsec_types::prelude::*;
+pub use recovery_device::{
+    import_recovery_device, ExportRecoveryDeviceError, ImportRecoveryDeviceError,
```

Comment:
I think it is a good idea to put this `import_recovery_device` in client (since the export part is there and the two are closely related) even if it is really related to `Client`

So I wonder if they would be a gain to expose `import_recovery_device` as a `Client` method anyway:
```rust
impl Client {
    [...]
    pub async fn import_recovery_device(
        &self,
       [...]
    ) -> Result<AvailableDevice, ClientImportRecoveryDeviceError> {
        recovery_device::import_recovery_device(self, [...]).await
    }
    [...]
}
```

The main gain I see is we are more consistent when naming the errors:
- Since we have `Client::export_recovery_device`, we should have`ClientExportRecoveryDeviceError`
- But if `import_recovery_device` is not a method of `Client`, it may be a bit weird to have in error named `ClientImportRecoveryDeviceError`
- And if its error is called `ImportRecoveryDeviceError`, then it is also weird since it this error is defined in `src/client` where all other public errors have a `Client` prefix (so it feels like an error in the naming...)

Why do you think ?

Reply:
We could do that, it justifies the client prefix, but it's kinda artificial :shrug: 
```diff

Reply:
You're right, this is totally artificial :/
On top of that, I've just realize it doesn't play nice with the functions exposed in `libparsec` where anything starting with `client_` should provide a client Handle (same thing for anything starting with `workspace_` should provide a workspace Handle)

So in the end we should just name it `import_revovery_device` & `ImportRecoveryDeviceError` and keep it as a standalone function
(but of course we still have `Client::export_recovery_device`, so the error is `ClientExportRecoveryDeviceError` and the binding function in `libparsec` is `client_export_recovery_device`)

Aaaaand sorry for all thus fuzz on such a boring topic :pray: